### PR TITLE
Minor changes and new war checker

### PR DESCRIPTION
**Pull request by aldent95**

### CODE REVIEW

File: backend/src/tdf/fitcheck.rs [hunk 1]
```diff
@@ -333,16 +336,16 @@ impl<'a> FitChecker<'a> {
                 } else if doctrine_fit.name.contains(&set_tag.to_title_case())
                     || (set_tag == "WARPSPEED"
                         && !(doctrine_fit.name.contains("Amulet")
-                            || doctrine_fit.name.contains("Hybrid")))
+                            || doctrine_fit.name.contains("Amulet")))
```
Comment (samuelgrant) [inline backend/src/tdf/fitcheck.rs]: Check this

File: backend/src/tdf/fitcheck.rs [hunk 2]
```diff
@@ -333,16 +336,16 @@ impl<'a> FitChecker<'a> {
...
+                    || (set_tag == "AMULET" && doctrine_fit.name.contains("Amulet"))
                 {
                     self.tags.insert(set_tag);
                     // give warning if you have all but slot 10 or wrong slot for that ship
                     if implantmatch::detect_slot10(self.fit.hull, self.pilot.implants).is_none() {
                         self.tags.insert("NO-SLOT10");
                     }
-                    if set_tag == "AMULET" && doctrine_fit.name.contains("Hybrid") {
+                    if set_tag == "AMULET" && doctrine_fit.name.contains("Amulet") {
                         self.tags.insert("SLOW");
```
Comment (samuelgrant) [inline backend/src/tdf/fitcheck.rs:348]: And this logic

File: frontend/src/App/Menu.js [hunk 3]
```diff
@@ -117,7 +117,7 @@ export function Menu({ onChangeCharacter, theme, setTheme, sticker, setSticker }
           <NavBar.Header>
             <MobileNavButton isOpen={isOpenMobileView} setIsOpen={setOpenMobileView} />
             <NavBar.LogoLink to="/">
-              <NavBar.Logo src={logoImage} alt="Contingency Incursions" />
+              <NavBar.Logo src={theme == "Light" ? logoLightImage : logoDarkImage} alt="Contingency Incursions" />
```
Comment (samuelgrant) [inline frontend/src/App/Menu.js:120]: Check CSS/Theme rules to ensure invert() is not used

File: frontend/src/Pages/Fits/FittingSortDisplay.js [hunk 4]
```diff
@@ -257,7 +257,7 @@ function ImplantOut() {
     <>
       <DisplayDOM style={{ justifyContent: "initial" }}>
         <ImplantButton name="Ascendancy" img={wbadge} />
-        <ImplantButton name="Hybrid" img={hbadge} />
+        <ImplantButton name="Amulet" img={hbadge} />
```
Comment (samuelgrant) [inline frontend/src/Pages/Fits/FittingSortDisplay.js:260]: Probably want to get rid of the H and replace with an A for amulate

File: frontend/src/App/Menu.js (hunk 3 above)
Comment (aldent95) [reply frontend/src/App/Menu.js:120]: Already done

File: frontend/src/Pages/Fits/FittingSortDisplay.js (hunk 4 above)
Comment (aldent95) [reply frontend/src/Pages/Fits/FittingSortDisplay.js:260]: Points to the hardwire image, so it's the same no matter which one it is.

**Pull request merged**